Cargo.toml

```diff
@@ -24,6 +24,7 @@ log = "0.3"
 env_logger = "0.3"
 toml = { version = "0.2", features = ["serde"] }
 open = "1.1"
+regex = "0.1.80"
```

Comment:
Updating deps is good, but maybe it's better if I first accept #206 then to avoid conflicts